src/components/browse/singleDatasetView/singleDatasetViewAccordion/tables/filesTable.ts

```diff
@@ -41,6 +41,11 @@ export const FilesTable = (props: FilesTableProps) => {
 
   const allFiles = props.allFiles || [];
 
+  const storageDict = [
+    { key: "TUE", value: "Tübingen" },
+    { key: "HD", value: "Heidelberg" },
+  ];
+
```

Comment:
This should be a constant mapping. You can already add the future data hubs.

```ts
const STORAGES: { [key: string]: string } = {
  "B": "Berlin",
  "DD": "Dresden",
  "HD": "Heidelberg",
  "K": "Cologne",
  "KI": "Kiel",
  "M": "Munich",
  "TUE": "Tübingen",
};
```

Also, you can move this out of the component to the top level of the file.